payment/payment.go

```diff
@@ -47,8 +53,14 @@ func FromPayment(payment *Payment) (*Payment, error) {
 }
 
 // FromPayment creates a nested Payment struct from script
-func FromScript(script []byte) (*Payment, error) {
-	redeem := &Payment{Script: script}
+func FromScript(net *network.Network, script []byte) (*Payment, error) {
```

Comment:
just for consistency, change the order of the arguments so that it's the same of the `FromPublicKey` function.
```suggestion
func FromScript(script []byte, net *network.Network) (*Payment, error) {
```

Reply:
Done.